### DISCUSSION TITLE
Why TuYa instead of Tuya?

### DISCUSSION THREAD

**Discussion by artem-dudarev** [Koenkk/zigbee2mqtt#22876] [Devices]
@Koenkk Why do you always write `TuYa` when the official website https://www.tuya.com spells this word as `Tuya`?

**Comment by Koenkk** [comment]
Should be `Tuya` indeed, renamed it!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.
